src/redux/backend-session.spec.ts

```diff
@@ -205,4 +206,32 @@ describe("epics", () => {
       store.getActions().find(action => action.type === startSession.type)
     ).toBeDefined();
   });
+
+  describe("selectImageTag", () => {
+
+    test("should return the python image tag based on version chosen", () => {
+      const language = "python";
+      const version5 = "3.5";
+
+      expect(selectImageTag(language, version5) === "dcl-python-3.5:latest");
+    });
+
+    test("should return python3.5 tag as default if version is not recognised", () => {
+      const language = "python";
+      const version1 = "0.0";
+      const version2 = "2.7";
+      
+      expect(selectImageTag(language, version1) === "dcl-python-3.5:latest");
+      expect(selectImageTag(language, version2) === "dcl-python-3.5:latest");
+    });
+
+    test("should return the empty string if language is not python", () => {
+      const r = "r";
+      const sql = "sql";
+      const version = "3.5";
+
+      expect(selectImageTag(r, version) === "");
+      expect(selectImageTag(sql, version) === "");
```

Comment:
Same comment, maybe pass directly the value instead of defining `r` and `sql` variables

Reply:
Yes, it makes sense, thanks